### DISCUSSION TITLE
Release v0.9.2

### DISCUSSION THREAD

**Discussion by BazzaCuda** [BazzaCuda/CustomMenu#4] [Announcements]
Windows 64-bit binaries.

Config window:
-  Added a "Help" link to the Wiki pages here on Github.
-  Added a "Feedback" link to the Discussion page here on Github.

Some websites are making this application directly downloadable from their website rather than from here on Github.
As a result, some users may not be aware of the help available from the Wiki pages here.
I have provided links from the top right of the Config window to make users aware of the help and to hopefully generate some feedback/discussion here on Github.

Tested on both Windows 10 and Windows 11, this stable near-production release is made available to obtain feedback from a wider audience.

Just unzip to a folder of your choice and run.

See the wiki [Getting Started](https://github.com/BazzaCuda/CustomMenu/wiki/Getting-Started) page for more details.

All comments and feedback welcome in the discussion section.

[CustomMenu_release_v0_9_2.zip](https://github.com/BazzaCuda/CustomMenu/files/10862632/CustomMenu_release_v0_9_2.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/BazzaCuda/CustomMenu/releases/tag/v0.9.2'>Release v0.9.2</a>.</em>

**Comment by WhiteWraith** [comment]
I'm having an issue with the program not displaying text properly. It's showing black text on a black background. Is there a fix?
![2023-06-18_144428](https://github.com/BazzaCuda/CustomMenu/assets/37280464/599c95b4-3dbd-4c47-bd8e-35749a9fafa4)

**Reply by WhiteWraith** [reply]
You are right, at this point it must be something specific to my PC. I cant figure out what tho. This isn't happening to any other programs. I'll let you know if I find out whats going on. Thanks.

**Comment by BazzaCuda** [comment]
That's interesting; and not very helpful. I shall investigate.
1. If you add an item to the menu tree on the left, does that also display the same way?
2. And what about when you then Save, Close and right-click the desktop; what does the menu look like?
3. Do you have Windows set to use a Dark Theme or are you using a particular color scheme?
4. Presumably, you're running this on Windows 11, are you?

**Reply by WhiteWraith** [reply]
1. Yeah looks the same.
2. Yep, still no change.
3. I have both dark and light themes, no change.
4. I am using Windows 11 dev channel build 25370zn release 23516-0802.
5. Hope this helps.

**Reply by BazzaCuda** [reply]
Very strange. Especially as the labels for those edit boxes are explicitly set to the Windows standard White and the tips to the standard Gray. 

I'll have to get a Windows 11 VM to test it out on.

In the meantime, if you think of anything which might be out of the ordinary with your color/theme setup/personalization, please let me know.

**Reply by BazzaCuda** [reply]
Just to confirm, I've run CustomMenu in a Windows 11 virtual machine downloaded from Microsoft and it all displays as expected.
![image](https://github.com/BazzaCuda/CustomMenu/assets/22550919/9f083607-cdc4-4c47-ba43-b99b9e8f29dc)

Currently, the problem would appear to be isolated to something specific to your setup.

I'm happy to investigate further if you think of anything it might be.
